cli_bot.py

```diff
@@ -282,17 +308,17 @@ def get_contact(self, *args: str) -> str:
 
     @input_error(error_msg_base="Command 'birthdays' failed")
     def birthdays(self, *args: str) -> str:
-        """Prepares all contacts to be outputted into console that have BD in a following week.
+        """Prepares all contacts to be outputted into console that have BD in a following number of days.
 
         Args:
-            args: Command doesn't expect any args, list should be empty.
+            args: Number of days.
 
         Returns:
             Command output.
         """
         command_output = ""
         if len(args) != 1:
-            raise CommandOperationalError (
+            raise CommandOperationalError(
                 "Warning: Command expect one argument: number of days. "
                 f"Received: {' '.join(args)}\n"
             )
```

Comment:
Can we add some more fine tuned message for cases when user provided not an integer as a number of days ? f. ex. :

```
try:
     days = int(args[0])
except ValueError:
     raise CommandOperationalError("Command expects a valid integer - number of days, please recheck your input.")